ovn/utilities/ovn-nbctl.c

```diff
@@ -419,6 +419,13 @@ Route commands:\n\
                             remove routes from ROUTER\n\
   lr-route-list ROUTER      print routes for ROUTER\n\
 \n\
+Policy commands:\n\
+  lr-policy-add ROUTER PRIORITY MATCH ACTION [NEXTHOP]\n\
```

Comment:
Just my 2 cents :)

a. Based on the schema, i believe that CLIs should look like following.

b. We should have CLIs for policy creation, deletion and list, these CLIs should not take ROUTER name/uuid as input.

c. We should have CLIs for attaching policies to ROUTERs, these CLIs should take router name/uuid and policy uuid as input.

====

This is just my opinion, please feel free to override it :)

Reply:
As we discussed, this is based on ACLs.